pkg/monitor/lease.go

```diff
@@ -74,17 +78,25 @@ func parseMonitorFile(buffer []byte) (*yamlVips, error) {
 		return nil, err
 	}
 
-	if vips.APIVip == nil {
-		err := fmt.Errorf("APIVip is missing from the yaml content")
+	if vips.APIVip == nil && vips.APIVips == nil {
+		err := fmt.Errorf("APIVip(s) missing from the yaml content")
 		log.Error(err)
 		return nil, err
-	} else if vips.IngressVip == nil {
-		err := fmt.Errorf("IngressVIP is missing from the yaml")
+	} else if vips.IngressVip == nil && vips.IngressVips == nil {
+		err := fmt.Errorf("IngressVIP(s) missing from the yaml")
 		log.Error(err)
 		return nil, err
 	}
 
-	log.Info(fmt.Sprintf("Valid monitor file format. APIVip: %+v. IngressVip: %+v", *vips.APIVip, vips.IngressVip))
+	// Convert old-style single vip configs to the dual vip format
```

Comment:
like this ^ in my other question but also "remove-me version XYZ when dnsvip is fully deprecated"